adb_shell/exceptions.py

```diff
@@ -115,11 +115,11 @@ class UsbReadFailedError(Exception):
 
     """
     def __init__(self, msg, usb_error):
-        super(UsbReadFailedError, self).__init__(msg)
+        super(UsbReadFailedError, self).__init__(msg, usb_error)
         self.usb_error = usb_error
```

Comment:
Is `self.usb_error` needed now that you're passing `usb_error` above?

Reply:
Yeah, the base `Exception` stashes it in `args` rather than with a semantic attribute. I left it there to avoid changing the exception's interface.